Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
     "cerk_port_unix_socket",
     "cerk_port_health_check_http",
     "cerk_port_mqtt",
+    "cerk_port_mqtt_mosquitto",
```

Comment:
add the new cerk component to

- [ ] the main readme
- [ ] add it to `check-readme.sh`
- [ ] describe the difference between the two mqtt components

Reply:
> describe the difference between the two mqtt components

I will do this in https://github.com/ce-rust/cerk/pull/73